sgtl/graph.py

```diff
@@ -330,6 +330,39 @@ def tensor_product(self, other_graph):
         return Graph(scipy.sparse.kron(self.adjacency_matrix(), other_graph.adjacency_matrix()))
 
 
+def hypercube_graph(d: int) -> Graph:
+    """
+    Construct the hypercube unweighted graph on :math:`2**d` vertices.
+    :param d: Degree of the hypercube graph.
+    :return: The hypercube graph on :math:`2**d` vertices, as a `Graph` object.
```

Comment:
The text inside the `:math:` environment will be rendered using latex markup in the library documentation. So it will render in a nicer way with `2^d` rather than `2**d`.